src/org/omegat/Bundle.properties

```diff
@@ -2169,7 +2152,6 @@ MATCHES_OPEN_PREFERENCES=TM Matching Options
 DICTIONARY_HIDE=Hide '{0}'
 
 STARTUP_ERRORBOX_TITLE=Error on application startup
-STARTUP_JAXB_LINKAGE_ERROR=Either update to the latest JDK 1.6, or remove the jaxb-*.jar files from the OmegaT /lib dir
```

Comment:
Last modified in 2022 by @Kazephil  and discussed the message mensions too old Java version.